addons/grafana/dashboards/pilot-dashboard.json

```diff
@@ -790,5 +1443,5 @@
   "timezone": "browser",
   "title": "Pilot Dashboard",
   "uid": "3",
-  "version": 3
+  "version": 5
```

Comment:
just curious, why is this 5 and not 4

Reply:
i think the version counter changes with each reload + edit. So, in this particular case, I edited it and saved it 5 times before saving it to disk.